CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.0
+
+- 
```

Comment:
"Fixed the dynamic_library loading scheme by deferring the File().exists() check until failure to enrich exceptions, rather than prevent them. This file check caused issues in flutter applications where the resolved current working directory doesn't match the executable path, causing loading issues"